ui/component/or-dashboard-builder/src/or-dashboard-tree.ts

```diff
@@ -28,7 +28,7 @@ const treeStyling = css`
     }
 `;
 
-@customElement("or-dashboard-tree")
+@customElement('or-dashboard-tree')
```

Comment:
I'm not a big fan of single quotes for strings...should introduce some linting requirements

Reply:
Yeah I looked it up once, and practiced it since.
Configured the linter to practice double quotes from now on 👍